catalog.html

```diff
@@ -118,7 +117,7 @@ <h2>Каталог товаров</h2>
               <img src="https://via.placeholder.com/14x14" width="14" height="14" alt="pach">
             </li>
             <li>
-              <a href="#"><img src="https://via.placeholder.com/358x578" width="358" height="578" alt="Изобраение товара"></a>
+              <a href="#"><img src="img/img-sedona 1.jpeg" width="358" height="578" alt="Sedona"></a>
```

Comment:
```suggestion
              <a href="#"><img src="img/sedona.jpeg" width="358" height="578" alt="Sedona"></a>
```
Это не очень названия. Почти в 99.10% нужно менять название с того, как оно экспортируется с фигмы
(не нужно всяких номеров и тд. И `img` тоже лучше не нужно, лишний шум)